apiml-security-common/src/main/java/org/zowe/apiml/security/common/auth/saf/SafResourceAccessDummy.java

```diff
@@ -0,0 +1,125 @@
+/*
+ * This program and the accompanying materials are made available under the terms of the
+ * Eclipse Public License v2.0 which accompanies this distribution, and is available at
+ * https://www.eclipse.org/legal/epl-v20.html
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Copyright Contributors to the Zowe Project.
+ */
+package org.zowe.apiml.security.common.auth.saf;
+
+import lombok.Builder;
+import lombok.Value;
+import org.springframework.security.core.Authentication;
+import org.yaml.snakeyaml.Yaml;
+
+import java.io.*;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+public class SafResourceAccessDummy implements SafResourceAccessVerifying {
+
+    private static final String SAF_ACCESS = "safAccess";
+    private static final String DEFAULT_FILE_LOCATION = "saf.yml";
+    private static final String DEFAULT_RESOURCE_LOCATION = "mock-saf.yml";
+
+    private Map<ResourceUser, AccessLevel> resourceUserToAccessLevel = new HashMap<>();
+
+    public SafResourceAccessDummy() throws IOException {
+        File file = getFile();
+        if (file.exists()) {
+            try (
+                FileInputStream fis = new FileInputStream(file);
+                BufferedInputStream bis = new BufferedInputStream(fis)
+            ) {
+                load(bis);
+            }
+        } else {
+            try (InputStream inputStream = this.getClass().getClassLoader().getResourceAsStream(DEFAULT_RESOURCE_LOCATION)) {
+                load(inputStream);
+            }
+        }
+    }
+
+    protected File getFile() {
+        return new File(DEFAULT_FILE_LOCATION);
+    }
+
+    public SafResourceAccessDummy(InputStream inputStream) {
+        load(inputStream);
+    }
+
+    @Override
+    public boolean hasSafResourceAccess(Authentication authentication, String resourceClass, String resourceName, String accessLevel) {
+        ResourceUser resourceUser = ResourceUser.builder()
+            .resourceClass(resourceClass)
+            .resourceName(resourceName)
+            .userId(authentication.getName())
+            .build();
+        AccessLevel currentLevel = resourceUserToAccessLevel.get(resourceUser);
+        if (currentLevel == null) return false;
+        return currentLevel.compareTo(AccessLevel.valueOf(accessLevel)) >= 0;
+    }
+
+    private void load(InputStream inputStream) {
```

Comment:
Could you please add some comment or rename the load and set methods so we better know what they are doing. For future maintenance.